apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -5776,3 +5797,124 @@ missing_malicious_snapshot_and_fp_based_on_it(Cfg0) ->
     shutdown_(I4, R5, Cfg),
     ok.
 
+close_solo_and_conflict(Cfg) ->
+    Debug = get_debug(Cfg),
+    #{ i := #{fsm := FsmI} = I
+     , r := #{fsm := FsmR} = R
+     , spec := #{ initiator := PubI
+                , responder := PubR } = Spec} = create_channel_([?SLOGAN|Cfg]),
+    ok = rpc(dev1, aesc_fsm, close_solo, [FsmI, #{}]),
+    rpc(dev1, aesc_fsm, upd_transfer, [FsmR, PubR, PubI, 2]),
```

Comment:
Could you match that this returns ok? Later you assume that you get a signing request for the responder so this must return ok